src/lib/y2partitioner/blk_device_restorer.rb

```diff
@@ -191,9 +191,17 @@ def copy_subvolumes(source)
     #
     # @see Device#copy_to
     #
+    # @raise [RuntimeError] if the device has more than one parent
+    #
     # @param source [Y2Storage::Device]
     # @return [Y2Storage::Device] device copied to the current graph
     def copy_device(source)
+      if source.to_storage_value.in_holders.size != 1
```

Comment:
Why not simply `source.parents.size != 1`?

Reply:
Copying&pasting blindly :(